src/devices/Rfid/LogInfo.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+namespace Iot.Device.Rfid
+{
+    /// <summary>
+    /// The log level
+    /// </summary>
+    public enum LogLevel
+    {
+        None = 0,
+        Info = 1,
+        Debug = 2
+    }
+
+    /// <summary>
+    /// Output where to log the information
+    /// </summary>
+    [Flags]
+    public enum LogTo
+    {
+        Console = 0b0000_00001,
+        Debug = 0b0000_0010
+    }
+
+    /// <summary>
+    /// Simple log class to help in debugging the communication
+    /// between the PN532 and the host
+    /// </summary>
+    public class LogInfo
+    {
+        public static LogLevel LogLevel { get; set; }
+
+        public static LogTo LogTo { get; set; } = LogTo.Console;
+
+        public static void Log(string toLog, LogLevel logLevel)
```

Comment:
I believe there is a way to get TextWriter or Stream from console - that would be a more robust input than enum

Reply:
I found useful to be able to select which output to use. I've been working with containers on RPI with this sensor and using Console was useful. While connected on my Windows machine, Debug was all perfect. I used as well both in one occasion, when debugging SPI on the RPI connected in SSH with VS to do the debug.